django_codemod/visitors/core.py

```diff
@@ -0,0 +1,11 @@
+from django_codemod.constants import DJANGO_110, DJANGO_20
```

Comment:
isort:

> ./django_codemod/visitors/core.py:1:1: I001 isort found an import in the wrong position

I dont see the problem.

Reply:
Indeed that's a bit confusing, but if you run isort, it fixes the file as they're supposed to be. I have it running each time I save the files in my editor, and this is the fixed version:

```suggestion
from django_codemod.constants import DJANGO_20, DJANGO_110
```

Basically isort checks for the whole number, and 110 > 20.

Reply:
Then that the tool forcing a bug, and confusing contributors.

To work with the tool, these constants should be named DJANGO_1_10, etc.

Or there should be some config for isort.

Reply:
Calling it a "bug" is a bit much 😄 That's merely a style annoyance.

I didn't know of this isort behaviour when defining these constants... I agree, we should rename them with an `_` in place of dots as you suggested, but IMOm this out of scope of this pull request.

Reply:
Do you want to do it, or should I (separate PR of course)?

Need to worry about backwards compatibility?  IMO it is unlikely external code is using them.
And none are [on GitHub](https://github.com/search?q=django_codemod&type=Code)

Reply:
It is a style annoyance, but one which bites newcomers, and IMO tool integration which bites newcomers  are pretty high priority problems.  I consider myself rather familiar with these tools, but this stopped me in my tracks as it wasnt obvious, which is silly because there are only two possible ways to order the symbols for this line, so I should have guessed why the other one was being requested, but the requested one is so illogical I couldnt make sense of it.

Reply:
Yes it's never great when this happens...

You can make this change if you want, that would be a great help. Otherwise I'll do it later when I find time.

I agree I don't think we need to worry about backwards compatibility here, it's very much internal stuff.